### PR TITLE
Ktl_XV's Dev Update 1

### PR DESCRIPTION
**Pull request by Ktl-XV**

### CODE REVIEW

File: development-updates.md [hunk 1]
```diff
@@ -8,31 +8,32 @@ Add your name **alphabetically** and link your updates to the corresponding tabl
...
+| [Hopinheimer](https://github.com/hopinheimer)                | [Update 0](https://hackmd.io/@np8VhkKRRHanpT1vbtRQ4Q/ByAoT58Q0)                                                                                  |                                                                 |        |     |     |     |
+| [Krishang](https://github.com/kamuik16)                      |                                                                                                                                                  | [Update 1](https://hackmd.io/@kamuik16/SyVy4aKrR)               |        |     |     |     |
+| [Ktl_XV](https://github.com/ktl-xv)                          |                                                                                                                                                  | [Update 1](https://hackmd.io/@Ktl-XV/EPF5-W1)                   |        |     |     |     |
```
Comment (Ktl-XV) [inline development-updates.md:30]: This is my actual change

Comment (taxmeifyoucan) [reply development-updates.md:30]: It's because the formatter you used created 3 extra columns. Please remove those first

Comment (Ktl-XV) [reply development-updates.md]: :facepalm: Fixed

Comment (taxmeifyoucan) [reply development-updates.md]: No problem, thanks for the update! Love the whiteboard brainstorming :)

**Pull request merged**